Doc_grammar: Update cmd:: and tacn:: constructs in .rsts

jfehrle opened the pull request:
See the README.md for further description.

This PR also includes many edits for commands and tactics for chapters that haven't been updated.  And 2 small rst changes.

jfehrle commented:
PS I plan to have all further syntax updates rely on this PR.

Zimmi48 commented on doc/sphinx/language/gallina-specification-language.rst:
```diff
@@ -1533,10 +1533,8 @@ Computations
+   occs_nums ::= {+ {| @num | @ident } }
+   | - {| @num | @ident } {* @int_or_var }
    int_or_var ::= @int
    | @ident
```
It's a bit surprising that you choose to inline the definition of `num_or_var` but not `int_or_var`.

jfehrle replied:
num_or_var was only used twice, both in occs_nums.  int_or_var is used 34 times, seemed less obvious.  BTW, at some point we may want to reconsider some inlinings if they save us from describing the same details over and over rather than having a hyperlink.

Easy to change these, of course.

Zimmi48 commented on doc/tools/docgram/README.md:
```diff
@@ -58,22 +58,26 @@ tactic_expr5: [
+place in `orderedGrammar`, therefore users should review the output and make
+appropriate adjustments to the order.
+
+The update process removes manually-added comments from `orderedGrammar` while
```
You can indent the content of your bullet-point to avoid new paragraph inside a bullet point to feel oddly unindented in the rendered version.

jfehrle commented:
> Are you intending to remove the productionlist functionality? What about the prodn.edit_mlg and productionlist.edit_mlg files?

Yes.  doc_grammar no longer inserts productionlists into the rsts.  prodn.edit_mlg was to allow editing productionlists and prodns differently, so that's not needd any more, either.

Zimmi48 commented:
>  The edits can just be in common.edit_mlg (which is not as descriptive a name any more since all edits are now common).

Then please remove the other files and possibly rename the only remaining one.

jfehrle commented:
Updated.  I removed the 2 files.  I forgot to copy the deletions from my other branch.  I'll rename common.edit_mlg later--it's harmless and there's plenty of cleanup to do on doc_grammar.  Plus renaming it now would make it harder for me to diff the file across a couple local branches.
